src/core_modules/capture-core/components/WidgetIndicator/WidgetIndicatorContent/sortIndicatorsFn.ts

```diff
@@ -0,0 +1,38 @@
+import { localeCompareStrings } from '../../../utils/localeCompareStrings';
+import type { WidgetData } from '../../WidgetFeedback/WidgetFeedback.types';
+
+export const sortIndicatorsFn = (a: WidgetData, b: WidgetData): number => {
```

Comment:
This function refactoring is too complex and brings little value; roll back all the changes. Only keep `export const sortIndicatorsFn = (a, b): number`